packages/office-ui-fabric-react/src/components/DetailsList/examples/DetailsList.CustomGroupHeaders.Example.tsx

```diff
@@ -1,13 +1,41 @@
+// @codepen
+
 import * as React from 'react';
 import { Link } from 'office-ui-fabric-react/lib/Link';
 import { DetailsList, IGroup, IGroupDividerProps } from 'office-ui-fabric-react/lib/DetailsList';
-import { createListItems, createGroups } from 'office-ui-fabric-react/lib/utilities/exampleData';
-import './DetailsListExample.scss';
+import { createListItems, createGroups, IExampleItem } from 'office-ui-fabric-react/lib/utilities/exampleData';
```

Comment:
You added @codepen but I'm not sure this import would work in a codepen.

Reply:
Hmph...removed the codepen tags for now, but I made this to properly fix the issue. https://github.com/OfficeDev/office-ui-fabric-react/pull/7648